webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -37,10 +37,10 @@
 
 package bootstrap.liftweb
 
-import net.liftweb.http._
-import net.liftweb.common._
-import net.liftweb.sitemap.{Menu, _}
-import net.liftweb.sitemap.Loc._
+import net.liftweb.http.*
+import net.liftweb.common.*
+import net.liftweb.sitemap.{Menu, *}
+import net.liftweb.sitemap.Loc.*
```

Comment:
be careful, your intellij broke everything. You need to set these options in settings (ctrl+alt+s)
![image](https://user-images.githubusercontent.com/44649/191533513-a874dc0b-82f9-41de-a6e0-3ca383b0e8df.png)